manifest.test.js

```diff
@@ -1,9 +1,42 @@
+/* eslint-disable no-new */
```

Comment:
How come you had to add this? I can't see any additions where eslint would warn about this

Reply:
It was using a constructor as a side effect, thrown up by [this line](https://github.com/ably/features/blob/5dcba3154506affa718602e6d600637e346c7f4e/manifest.test.js#L34).